cmd/wnode-status/main.go

```diff
@@ -9,30 +9,6 @@ import (
 	"github.com/status-im/status-go/geth/params"
 )
 
-var (
-	prodMode    = flag.Bool("production", false, "Whether production settings should be loaded")
```

Comment:
Could you clarify, why do now we need 
 -	nodeKeyFile = flag.String("nodekey", "", "P2P node key file (private key)")
 -	echo           = flag.Bool("echo", true, "Echo mode, prints some arguments for diagnostics")
 -	bootstrap      = flag.Bool("bootstrap", true, "Don't actively connect to peers, wait for incoming connections")
 -	forward        = flag.Bool("forward", false, "Only forward messages, neither send nor decrypt messages")
 -	port           = flag.Int("port", params.WhisperPort, "Whisper node's listening port")
?

Reply:
We don't need these flags.
* `nodeKeyFile` is to provide a custom private key for p2p communication but I haven't seen any usage of it or the purpose,
* `echo` it was blindly copied from go-ethereum `wnode` where it was used to debug purposes,
* `bootstrap` also copied from go-ethereum used for some debug purposes,
* `forward` again copied from go-ethereum, it disables stdin,
* `port` unused flag. It's replaced with `listenAddr`.

Reply:
`bootstrap` - is useful for mailserver to not to get messages from all network and listen only few peers.

Reply:
@JekaMas I can see it useful during tests but not sure how is it useful with MailServer. MailServer must support all Status app instance and you can not add them beforehand as individual peers.